agent/engine/engine_integ_test.go

```diff
@@ -507,9 +507,11 @@ func TestLabels(t *testing.T) {
 	assert.EqualValues(t, "", state.Config.Labels["label1"])
 
 	// Kill the existing container now
-	taskUpdate := *testTask
+	// Create instead of copying the testTask, to avoid race condition.
+	// AddTask idempotently handles update, filtering by Task ARN.
```

Comment:
nit: this two-line comment seems like it should be in the commit message, not a comment in the code. Someone reading this won't really have the context that there used to be a 'copy' operation being performed here

Reply:
I think the comment still makes sense without the context, as I'd be wondering why are we creating new test task even if I didn't know we switched from copy to create.